packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -356,6 +358,54 @@ class VifStatus extends BaseComponent {
   }
 }
 
+const AdvancedSettingsModal = decorate([
+  provideState({
+    effects: {
+      toggleTxChecksumming() {
+        const { onChange, value } = this.props
+        onChange({
+          ...value,
+          txChecksumming: !value.txChecksumming,
+        })
+      },
+    },
+  }),
+  injectState,
+  ({ effects, state, value }) => (
+    <Container>
+      <SingleLineRow>
+        <Col mediumSize={6}>
+          <strong>{_('txChecksumming')}</strong>
+        </Col>
+        <Col mediumSize={6}>
+          <StateButton
+            className='pull-right'
+            disabledLabel={_('stateDisabled')}
+            enabledLabel={_('stateEnabled')}
+            handler={effects.toggleTxChecksumming}
```

Comment:
We can use `toggleState`?

Reply:
No, because it only supports native inputs. See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/reaclette-utils.js#L6